api/graphql/schema.graphql

```diff
@@ -391,21 +399,26 @@ type Department {
   teams: [Team] @belongsToMany @canModel(ability: "viewAny")
 }
 
-type Community {
+type Community implements HasRoleAssignments {
   id: ID!
   key: String!
   name: LocalizedString
   description: LocalizedString
+  team: Team @morphOne @canModel(ability: "viewAny")
+  roleAssignments: [RoleAssignment!]
+    @hasManyThrough
+    @canRoot(ability: "viewTeamMembers")
 }
 
-type Team {
+type Team implements HasRoleAssignments {
```

Comment:
I can't really get anything out of the queries.  As soon as I add roleAssignments I start getting unauthorized exceptions.  Is is that there are no teams seeded for the pools or communities yet?

Reply:
Are you still seeing this with the new seeding?